projects/libyaml/libyaml_emitter_fuzzer.c

```diff
@@ -24,26 +24,28 @@
 #undef NDEBUG
 #endif
 
-#define BUFFER_SIZE 65536
 #define MAX_EVENTS 1024
 
+typedef struct yaml_output_buffer {
```

Comment:
can we move this struct & the handler to a header/c file and include it?
duplicating this is not so nice...

Reply:
Sure, SGTM.